doc/perception/README.md

```diff
@@ -1,26 +1,36 @@
 # Documentation of perception component
 
-This folder contains further documentation of the perception components.
-
-1. [Vision Node](./vision_node.md)
-   - The Visison Node provides an adaptive interface that is able to perform object-detection and/or image-segmentation on multiple cameras at the same time.
-2. [Position Heading Filter Debug Node](./position_heading_filter_debug_node.md)
-3. [Kalman Filter](./kalman_filter.md)
-4. [Position Heading Publisher Node](./position_heading_publisher_node.md)
-5. [Distance to Objects](./distance_to_objects.md)
-6. [Traffic Light Detection](./traffic_light_detection.md)
-7. [Coordinate Transformation (helper functions)](./coordinate_transformation.md)
-8. [Dataset Generator](./dataset_generator.md)
-9. [Dataset Structure](./dataset_structure.md)
-10. [Lidar Distance Utility](./lidar_distance_utility.md)
-    1. not used since paf22
-11. [Efficient PS](./efficientps.md)
-    1. not used scince paf22 and never successfully tested
-
-## Overview Localization
+This folder contains documentation of the perception components.
+
+In the following diagram you can see all the nodes and topics used in the perception to show how they work together.
+
+![Perception Overview](../../doc/assets/perception/perception_overview.png)
+
+## Object Detection / Distance and Segmentation
+
+- [Vision Node](./vision_node.md)
+  - The Vision Node provides an adaptive interface that is able to perform object detection and/or image segmentation on multiple cameras at the same time.
```

Comment:
What does "and/or" mean in this case? Are there restrictions to the simultaneous execution?

Reply:
I just copied this summary sentence from the "old" README file. The other nodes don't get a summary so I wasn't sure if I should leave it in but as the vision_node is quite important I thought a short summary might be nice. I think some models that can be selected in the node can only do object detection or image segmentation while others can do both, right? I think that's why "and/or" was used.

Reply:
As of now, the vision node is capable of providing object detection via the NN with YOLO/Ultralytics and image segmentation for the traffic lights. As far as I know. So I would write 'and'. 
But at the same time, I would clarify, that at the current state it is not possible to use multiple cameras (correct me if I'm wrong)